src/main/java/oshi/software/os/mac/local/CentralProcessor.java

```diff
@@ -32,8 +37,14 @@
 	 * @return Processor vendor.
 	 */
 	public String getVendor() {
-		if (_vendor==null)
-			_vendor=ExecutingCommand.getFirstAnswer("sysctl -n machdep.cpu.vendor");
+		if (_vendor == null) {
+			int[] mib = { SystemB.CTL_MACHDEP, SystemB.MACHDEP_CPU,
+					SystemB.MACHDEP_CPU_VENDOR };
+			IntByReference size = new IntByReference(100);

```

Comment:
Yes, I believe you can call once sending null for the pointer to get the size, and call again.  I'll address this.  I found that constant 100 in some other Apple code, which isn't exactly documentation but it's an example... moot point if I do two calls.